corehq/apps/app_manager/templates/app_manager/v2/form_view_base.html

```diff
@@ -211,7 +211,7 @@
 
         gaTrackLink($('.breadcrumb .view-form'), 'App Builder', 'View Form', 'Breadcrumb');
 
-        {% if allow_cloudcare %}
+        {% if can_preview_form and allow_cloudcare %}
```

Comment:
If Manish tests locally in v2 (turn on `Prototype for case management onboarding (App Manager V2)`) and verified it looks all right, that's good due diligence.